fcrepo-http-api/src/main/resources/views/common-node-actions.vsl

```diff
@@ -50,13 +50,23 @@
     <input type="text" id="new_id" placeholder="(auto-generated identifier)" name="slug"  class="form-control"/>
         </div>
 
+    <div class="form-group" id="turtle_payload_container">
+        <label for="rdf"  class="control-label">
+	        RDF (turtle)
+        </label>
+        <textarea rows="10" id="turtle_payload" name="turtle" class="form-control" 
+                  placeholder="@prefix dcmitype: <http://purl.org/dc/dcmitype/> 
+<> a dcmitype:Collection ."></textarea>
```

Comment:
Another point here -- but Firefox (I believe) doesn't allow newlines in attributes (it just ignores them). So, I wonder if that implies anything about the example provided.